src/views/FAQ/deleteFAQ.js

```diff
@@ -0,0 +1,121 @@
+import { toast, ToastContainer, Zoom } from 'react-toastify';
+import { QuestionContext } from '../../contexts/QuestionContext';
+import DashboardLayout from '../../components/DashboardLayout/DashboardLayout';
+
+const { REACT_APP_BACKEND_URL } = config;
+const userRoleId = localStorage.getItem('userId');
+console.log(userRoleId);
+
+const DeleteFAQ = () => {
+  const classes = useStyles();
+  const { questions, setQuestions } = useContext(QuestionContext);
+  
+  useEffect(() => {
+    const getQuestions = async () => {
+      try {
+        const response = await axios.get(`${REACT_APP_BACKEND_URL}/questions`);
+        
+        setQuestions(response.data.allQuestions);
+      } catch (error) {
+        toaster(error, 'Internal server error');
+      }
+    };
+    
+    getQuestions();
+  }, [setQuestions])
+  
+  const handleDelete = async (id) => {
+    try {
+      await axios.delete(`${REACT_APP_BACKEND_URL}/questions/${id}`, {
+        headers: {
+          "Content-Type":"Application/json",
+          'authorization': localStorage.getItem('userToken')
+        }
+      });
+      
+      setQuestions(questions.filter((question) => question.id !== id));
+      
+      toaster('Question deleted successfully!', 'success');
+    } catch (error) { 
+      toaster('Internal server error', 'error');
+    }
+  }
+  return (
+    <DashboardLayout>
+      <ToastContainer 
+          draggable={true}
+          transition={Zoom}
+          autoClose={3000}
+          position={toast.POSITION.TOP_CENTER}
+      />
+          <div className={classes.faqContainer}>
+            <div className={classes.searchContainer}>
+                <Typography variant="h6" className={classes.title} style={{ textAlign: 'center' }}>
+                  Frequently Asked Questions
+                </Typography>
+            </div>
+            <div className={classes.questionContainer}>
+              <div className={classes.leftDivider}>
+                <div className={classes.contentContainer}>
+                  {questions.map((question, index) => (
+                    <div key={question.id} className={classes.content} style={{ display: (index + 1)%2 !== 0 ? 'block' : 'none' }}>
+                      <Badge badgeContent={index + 1} color="primary" classes={{badge: classes.badge}}/>
+                      <Typography variant="h6" className={classes.title}>{question.subject}</Typography>
+                      <Typography className={classes.body}>
+                        {question.message.slice(0, 250) + '...'}
+                      </Typography>
+                      <div style={{ display: 'flex', justifyContent: 'space-between', width: '96%' }}>
+                        <Typography className={classes.body} style={{ margin: '20px 0 0 10px' }}>
+                          Asked by <span className={classes.infos}  style={{ color: '#127CDD' }}>{question.name}</span> on <span className={classes.infos}>{question.createdAt}</span> 
```

Comment:
Can change this date `toLocaleDateString()` to readable well instead of those time zone

Reply:
Resolved!